openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -3343,7 +3354,7 @@ def check_handler_compatibility(self, other_handler):
             tolerance_attrs=float_attrs_to_compare,
         )
 
-    def find_matches(self, entity):
+    def find_matches(self, entity, unique=False):
```

Comment:
(blocking) (1) The `unique` kwags isn't used in this method. I think it needs to be added.